src/components/sidebar.py

```
@@ -0,0 +1,97 @@
+    st.sidebar.caption(f"探索範囲: {settings['k_min']:,} 〜 {settings['k_max']:,}")
+
+    # 成長率 (γ)
+    st.sidebar.subheader("成長率 (γ) の範囲")
+    gamma_min = st.sidebar.number_input(
+        "最小値 (γ_min)", value=0.0285, min_value=0.0, step=0.0001, format="%.4f"
+    )
+    gamma_max = st.sidebar.number_input(
+        "最大値 (γ_max)", value=0.0350, min_value=0.0, step=0.0001, format="%.4f"
+    )
+    gamma_step = st.sidebar.number_input(
+        "ステップ (γ_step)", value=0.0005, min_value=0.0, step=0.0001, format="%.4f"
```

Ruff (0.12.2)

String contains ambiguous `γ` (GREEK SMALL LETTER GAMMA). Did you mean `y` (LATIN SMALL LETTER Y)?

(RUF001)